docs/source/install/install.rst

```diff
@@ -89,20 +89,21 @@ Linux & OSX
    # optional
    ctest
 
-   # sudo might be required required for system paths
+   # sudo might be required for system paths
    cmake --build . --target install
 
 Windows
 ^^^^^^^
 
-Replace the last commands with:
+The process is basically similar to Linux & OSX, with just a couple of minor tweaks. Use ``mkdir openPMD-api-build`` without the ``-p`` option and replace the last three commands with
```

Comment:
Thanks!
In order to simplify, can you please:
- start a new line with the second sentence
- remove `-p` from the Linux example, it's not needed and ok to fail if existing dir is in place
- replace "Use ``mkdir openPMD-api-build`` without the ``-p`` option" with "Use ``ps ..\openPMD-api\.travis\download_samples.ps1`` to download sample files for tests"